src/echo/echo.rs

```diff
@@ -3,7 +3,7 @@
 /*
  * This file is part of the uutils coreutils package.
  *
- * (c) Derek Chiang <derekchiang93@gmail.com>
+ * (c) Christopher Brown <ccbrown112@gmail.com>
```

Comment:
I'd prefer if you added your name instead of replacing the original author's name.